Gratwurst.lua

```diff
@@ -277,6 +258,16 @@ local function slashcmd(msg, editbox)
 	elseif (msg == "disable") then
 		GratwurstEnabled = false;
 		print("Gratwurst disabled.");
+	elseif (msg == "debug") then
+		-- output saved variables to chat
+		print("=======================")
+		print("GratwurstDelayInSeconds: " .. GratwurstDelayInSeconds)
+		print("GratwurstRandomDelayMax: " .. GratwurstRandomDelayMax)
+		print("GratwurstEnabled: " .. tostring(GratwurstEnabled))
+		print("GratwurstRandomDelayEnabled: " .. tostring(GratwurstRandomDelayEnabled))
+		print("GratwurstShouldVary: " .. tostring(GratwurstShouldVary))
+		print("GratwurstVariancePercentage: " .. GratwurstVariancePercentage)
+		GuildAchievementMessageEventReceived(true);
```

Comment:
[?] Is this intended to be a function call?  It seems like this might be the duplicate calls being made.  I don't see where Log is called though, so maybe I'm wrong?

Reply:
Never mind, I realized this is intended to display in the debug window, and doesn't affect the rest of the addon.